tests/subsys/zigbee/osif/serial/serial_via_logger/testcase.yaml

```diff
@@ -1,5 +1,5 @@
 tests:
-  zigbee.osif.logger:
+  zigbee.osif.logger.serial_via_logger:
```

Comment:
`logger` appears twice. Maybe the first `.logger` is redundant?

Reply:
Done